app-frontend/src/assets/styles/sass/components/_pagination.scss

```diff
@@ -0,0 +1,96 @@
+$pagination-font-size:          14px !default;
```

Comment:
I wouldn't worry too much about styling issues right now. You can lean on me for that. I'll be doing a pass through styles once it's merged in.